docs/java/sniffer.asciidoc

```diff
@@ -0,0 +1,126 @@
+== Sniffer
+
+Minimal library that allows to automatically discover nodes from a running
+Elasticsearch cluster and set them to an existing `RestClient` instance.
+It retrieves by default the nodes that belong to the cluster using the
+Nodes Info api and uses jackson to parse the obtained json response.
+
+Compatible with Elasticsearch 2.x and onwards.
+
+=== Maven Repository
+
+Here is how you can configure the dependency using maven as a dependency manager.
+Add the following to your `pom.xml` file:
+
+[source,xml]
+--------------------------------------------------
+<dependency>
+    <groupId>org.elasticsearch.client</groupId>
+    <artifactId>sniffer</artifactId>
+    <version>${es.version}</version>
```

Comment:
Ah ok, in that case you need to define `es.version` in `index.asciidoc` (i'll add a separate comment there).

Then you need to change the code block to:

    ["source","xml",subs="attributes"]
    --------------------------------------------------
     <dependency>
         <groupId>org.elasticsearch.client</groupId>
         <artifactId>sniffer</artifactId>
         <version>{version}</version>   
     </dependency>
     --------------------------------------------------